src/import/export.c

```diff
@@ -25,6 +25,7 @@
 
 static ImportCompressType arg_compress = IMPORT_COMPRESS_UNKNOWN;
 static ImageClass arg_class = IMAGE_MACHINE;
+static RuntimeScope arg_runtime_scope = _RUNTIME_SCOPE_INVALID;
```

Comment:
as mentioned in the git commit, this is supposed to be hooked up sooner rather than later. i didn't want to add more stuff to the PR that ports importd fully over

but it seemed nicer to me to prep the groundwork here so that we only have to add the cmdline parsers later.

Reply:
@YHNdnzj `_RUNTIME_SCOPE_INVALID` means both system and user.

Reply:
> as mentioned in the git commit, this is supposed to be hooked up sooner rather than later. i didn't want to add more stuff to the PR that ports importd fully over
> 
> but it seemed nicer to me to prep the groundwork here so that we only have to add the cmdline parsers later.

Maybe add a TODO then?

Reply:
added some entries to TODO now